metrics/src/test/java/com/expedia/adaptivealerting/metrics/functions/source/MetricFunctionsReaderTest.java

```diff
@@ -15,8 +15,9 @@
 
     @Test
     public void testReadFromInputFile(){
-        val functionInputFileName = "/config/functions-test.txt";
-        List<MetricFunctionsSpec> metricFunctionsSpecList = MetricFunctionsReader.readFromInputFile(functionInputFileName);
+        val functionInputFileName = "config/functions-test.txt";
+        List<MetricFunctionsSpec> metricFunctionsSpecList = MetricFunctionsReader.readFromInputFile
+                (ClassLoader.getSystemResource(functionInputFileName).getPath());
```

Comment:
same as above

Reply:
Deployment uses absolute path of file - "/config/functions.txt". 
Updated test cases to reflect the same as well.